### DISCUSSION TITLE
Documentation suggestion for new users

### DISCUSSION THREAD

**Discussion by Iansutt** [helix-editor/helix#10649] [General] [answered]
New user here and a really simple add for documentation is mention for relevant installs. Mention that hx is the launch command for the application. I don't want to admit how long it took me to find this. i happened to stumble upon the arch install section and it's mentioned there but nto for my distro (NixOS) and i imagine would be relevant for the other distros. Simple change. and sorry if i missed something obvious but i did all the Ctrl-F's i could think of.

**Comment by TornaxO7** [comment, marked as answer]
> (...) and it's mentioned there but nto for my distro (NixOS) (...)

![image](https://github.com/helix-editor/helix/assets/50843046/c51f8fb1-d8ae-4c3f-92fd-3ff93c6e296b)

Also it's mentioned in the [`Usage section`](https://docs.helix-editor.com/usage.html) to use `hx` although to be fair it may be not completely clear.

**Comment by Iansutt** [comment]
totally fair, When searching the usage page i did not put that together but in hindsight I can see it.
